style.css

```diff
@@ -0,0 +1,18 @@
+h1 {
+  font-family:Merriweather;
```

Comment:
I would put a space between the `:` and its property to break up the key/value pair and make it a little more readable.